Orhan_JS2_Week1_Homework/style.css

```diff
@@ -0,0 +1,30 @@
+
+.book_list ul {
+  padding: 0px;
+}
+.book_list li {
+  float: left;
+  position: relative;
+  list-style: none;
+  color: blue;
+  border: 1px solid black;
+  margin: 10px;
+  padding: 10px;
+  width: 400px;;
```

Comment:
Hard-coding width in pixels works here, but means that it is less flexible if you are looking at this page with a phone or tablet.